docs/conf.py

```diff
@@ -9,7 +9,9 @@
 
 # django-specific configuration
 from django.conf import settings
-settings.configure()
+
+# Removed for breaking doc building with django-guardian installed.
+# settings.configure()
```

Comment:
It blew up when building docs because `ANONYMOUS_USER_ID` did not exist in settings. It _is_ in `lore/settings.py`, but `settings.configure()` blows away the settings with all defaults. I left that line and the comment in just in case someone else knew a better solution to the problem.